src/sentry/static/sentry/app/utils.tsx

```diff
@@ -276,3 +276,17 @@ export function generateQueryWithTag(
 }
 
 export const isFunction = (value: any): value is Function => typeof value === 'function';
+
+export function filterProps<T extends {}, K extends keyof T>(
```

Comment:
Is this not similar to lodash/pick?

Reply:
I believe it's similar to `lodash/omit `, when it comes to simple operations, I always prefer to use the native function than lodash...but I've replaced the filterProps 😉